test/viewmodels/dm/test_notes_section.js

```diff
@@ -7,40 +7,40 @@ describe('NotesSectionViewModel', function(){
     });
 
     describe('Init', function() {
```

Comment:
In a lot of these tests (init/load/unload) you've removed the success criteria. After the to-be-tested method is run, there's nothing to show that it actually worked as expected... What is the reason you removed this?